infrastructure/server-setup/playbook-1.yml

```diff
@@ -83,6 +83,51 @@
       script: ../cryptfs/mount.sh -p {{encrypt_passphrase}}
       when: st.stat.exists and encrypt_data
 
+    # https://stackoverflow.com/a/24765946
```

Comment:
## [Automatically allocate 4GB of swap to every node created by Ansible playbooks](https://github.com/opencrvs/opencrvs-core/issues/2691)